README.md

```diff
@@ -1,19 +1,20 @@
-
 # API Project: Timestamp Microservice for FCC
 
 ### User stories :
 
-1. The API endpoint is `GET [project_url]/api/timestamp/:date_string?`
```

Comment:
Any reason for removing the '?' character. This could be used to make the :date_string parameter optional and those make the implementation simple.

Reply:
I thought the question mark is supposed to impose that preceding parameters are optional. I did not know that it also means that date_string can be optional. I will add the question mark back again.